Makefile

```diff
@@ -1159,68 +1161,67 @@ lint-md-clean:
 lint-md-build:
 	$(warning "Deprecated no-op target 'lint-md-build'")
 
-LINT_MD_DOC_FILES = $(shell ls doc/*.md doc/**/*.md)
-run-lint-doc-md = tools/lint-md.js -q -f $(LINT_MD_DOC_FILES)
+run-lint-doc-md = tools/lint-md.js -q -f $?
+
+# The following two targets are split for two reasons:
+# 1. The first one uses $(wildcard) and the second shells out 'find'.
+# 2. The first one uses 'available-node-shell' and the second uses
+#    $(call available-node, ...), we use that for debugging those two methods.
+
 # Lint all changed markdown files under doc/
-tools/.docmdlintstamp: $(LINT_MD_DOC_FILES)
-	@echo "Running Markdown linter on docs..."
-	@$(call available-node,$(run-lint-doc-md))
-	@touch $@
+tools/.docmdlintstamp: AVALIBLE_NODE := $(available-node-shell)
+tools/.docmdlintstamp: $(wildcard doc/*.md doc/**/*.md)
```

Comment:
`doc/**/*.md` doesn't appear to pick up all subdirectories under `doc`.

Reply:
5 mins of searching, and I couldn't even find a definition of what `pattern` means for `wildcard()` in GNU make, much less generally. I didn't think `**` was supported unless the shell supported it, and I just  tried on AIX, which is agressively POSIX, and it didn't expand.